extensions/sql-database-projects/package.json

```diff
@@ -186,6 +199,11 @@
           "when": "view == dataworkspace.views.main",
           "group": "1_currentWorkspace@1"
         },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+          "when": "view == dataworkspace.views.main",
```

Comment:
is there a default project when launched from command palette? What if there aren't any projects open in the workspace?

Reply:
This command is disabled for the command palette.  Having said that, it'd be completely fine to launch from there; the dialog can seeded with any combination (or lack) of database and project.